### DISCUSSION TITLE
zbInfo gone?

### DISCUSSION THREAD

**Discussion by rebel7580** [arendst/Tasmota#14542] [Q&A] [answered]
Hi -

It may be that what is missing is some of my brain cells, but I can't find zbInfo in either the Commands list for Zigbee, or in the zigbee page.  Did it disappear from Tamsota?  I saw no mention on discord or anywhere else of it going away.

Right now my Sonoff based bridge is running 10.0.0.3.

The reason I wanted to re-look at zbInfo documentation is that if I enter zbInfo, I get a list of only 11 of my 15 devices.  I had been using zbInfo to get the list of connected devices, but now that doesn't seem to be working, as it doesn't get the full list.  

Should I be using zbStatus instead?

**Comment by sfromis** [comment]
Well, the documentation can sometimes be "a bit spotty". ZbInfo should be just fine to use, was introduced in november 2020. Output seems to be basically equivalent to ZbStatus3. Is something "different" about the devices you miss in ZbInfo output?

You could try upgrading to newest Tasmota.

**Reply by rebel7580** [reply]
I could have sworn that ZbInfo was there in the docs at one point. Maybe an edit accidently lost it?

The four that are missing are an Aqara water leak detector,  an Aqara button, and two Jasco switches. These had shown up before.  The two switches are repeaters, and one each of the other two is linked to one of the switches. The switches have other devices connected to them and they show up. My four "newest" devices (paired most recently) show up. Almost seems like they pushed the others out. :)

If I do a``` zbinfo <device>```, I get a normal zbInfo return.

Wondering if there is a buffer issue somewhere.

**Comment by rebel7580** [comment]
Re: the missing ZbInfo in the docs:

Looking at the commits for Commands.md,  there were two commits on 4 Jan 2021. ZbInfo shows up in one of them, but is missing in the other. However, it's strange that in the commit where it is missing, it doesn't show that as a change. It's just missing.

ZbInfo line present:

Update Commands.md
Clarify and standardize SequenceOffset entry
 master (#578)
@pcdiem
pcdiem committed on Jan 4, 2021 
1 parent ddcf431 commit 10dd02c37ae69594dec69cab18aca64dd5c9ff56

```
ZbForget<a class="cmnd" id="zbforget"></a>|Used for devices that are unused but still visible in `ZbStatus`<BR>`<device>` = Remove a device from the Tasmota flash memory. It does not un-pair the device nor deleting the device information in the CC2530/ZBBridge.
ZbLeave<a class="cmnd" id="zbleave"></a>|`<device>` = request a device to leave the network
ZbInfo<a class="cmnd" id="zbinfo"></a>|`<device>` = display all information known about a device, equivalent to `ZbStatus3` with a simpler JSON output
ZbLight<a class="cmnd" id="zblight"></a>|Sets or reads the light type to be emulated in Zigbee Hue Emulation with Alexa.<BR>`<device>,<light_type>` sets the light type using an integer `0..5` corresponding to the number of channels (from one channel (on/off) to 5 channel (RGBCCT) lights)<BR>`<device>,-1` removes the device from Philips Hue emulation<BR>`<device>` displays the current status of the Light (Zigbee2Tasmota tracks all changes to the light)
ZbListen&#60;x> &#60;group>|(EZSP ZBBridge only)Listens to a multicast group address. By default EZSP will not report group messages unless you subscribe to the group.<BR>`<x>`: slot in the array of group addresses, 1..15<BR>`<group>`: group address to listen to, 0..0xFFFF<BR>At start-up, Z2T automatically listns to groupe 0 in slot 0.<BR>CC2530 does not need this command and always report all group messages.
```
Missing ZbInfo line:

Ruletimer0
 master (#579)
@barbudor
barbudor committed on Jan 4, 2021 
1 parent 6417e91 commit 6bb7b04ca402ac884a7b688e8c279fd6100ae560

```
ZbForget<a class="cmnd" id="zbforget"></a>|Used for devices that are unused but still visible in `ZbStatus`<BR>`<device>` = Remove a device from the Tasmota flash memory. It does not un-pair the device nor deleting the device information in the CC2530/ZBBridge.
ZbLight<a class="cmnd" id="zblight"></a>|Sets or reads the light type to be emulated in Zigbee Hue Emulation with Alexa.<BR>`<device>,<light_type>` sets the light type using an integer `0..5` corresponding to the number of channels (from one channel (on/off) to 5 channel (RGBCCT) lights)<BR>`<device>,-1` removes the device from Philips Hue emulation<BR>`<device>` displays the current status of the Light (Zigbee2Tasmota tracks all changes to the light)
ZbListen&#60;x> &#60;group>|(EZSP ZBBridge only)Listens to a multicast group address. By default EZSP will not report group messages unless you subscribe to the group.<BR>`<x>`: slot in the array of group addresses, 1..15<BR>`<group>`: group address to listen to, 0..0xFFFF<BR>At start-up, Z2T automatically listns to groupe 0 in slot 0.<BR>CC2530 does not need this command and always report all group messages.
```
ZBLeave also seems to have disppeared here.

I'l be the first to admit I don't fully understand how commits work in Gethub, but this seems strange.

**Comment by rebel7580** [comment]
W.r.t ZbInfo itself, the same issue occurs in the lastest tasmota-zigbee.bin.gz dev release 2022.01.3 20220121 21:17

Only 11 of my 15 devices are returned. Should this be spawned to an issue?

**Reply by s-hadinger** [reply]
Are you looking through the web console? If so, it's possible that you have exhausted the internal log buffer size. The MQTT or Serial outputs should be complete.

**Reply by rebel7580** [reply]
Well I feel dumb now. Yes, the console reply is the issue, I can see that the MQTT messages are all there. I should have looked at that in the first place.  Anyway, the docs look better now!  Thanks!

**Comment by s-hadinger** [comment, marked as answer]
I have copied back `ZbInfo` and `ZbLeave` in the documentation page. I don't know why or how they were removed.
